halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/local/git/LocalGitDeckService.java

```diff
@@ -128,4 +148,12 @@ public void commitWrapperScripts() {
         .setScript(script)
         .commitScript(Paths.get(getScriptsDir(), getArtifact().getName() + "-stop.sh"));
   }
+
+  protected Optional<String> customProfileOutputPath(String profileName) {
+    if (profileName.equals(deckSettingsLocalPath)) {
+      return Optional.of(homeDotSpinnakerPath + "settings-local-ignored.js");
```

Comment:
Is it worth doing this, or should we just ignore it completely and return empty?